bookwyrm/templates/layout.html

```diff
@@ -13,8 +13,9 @@
     <link rel="search" type="application/opensearchdescription+xml" href="{% url 'opensearch' %}" title="{% blocktrans with site_name=site.name %}{{ site_name }} search{% endblocktrans %}" />
 
     <link rel="shortcut icon" type="image/x-icon" href="{% if site.favicon %}{% get_media_prefix %}{{ site.favicon }}{% else %}{% static "images/favicon.ico" %}{% endif %}">
-    <link rel="apple-touch-icon" href="{% if site.logo %}{{ media_full_url }}{{ site.logo }}{% else %}{% static "images/logo.png" %}{% endif %}">
-
+    <link rel="apple-touch-icon" href="{% static "images/logo.png" %}">
```

Comment:
I think this should still keep the `site.logo` check for when an instance has configured a custom logo, no?

Reply:
If it is a .jpg or anything else than png the touch.icon will not work. Apple Touch Icon only works with png-files.

Reply:
Ah interesting, didn't know. I'm not sure it's a great user experience to choose the default one regardless of the customization on the instance. There might be a different solution to this

Reply:
Yeah, if we convert every uploaded logo to a png. Or create a separate smaller files as png. BTW: If the logo is not square the touch icon will make not to a square - no matter how it looks like. THAT is a bad user experience.

Reply:
You're right, but admin's can fix that by uploading a square logo. Only referencing the default image doesn't give admins any workaround to get their own logo into this.

Just tested it with a JPG and it does work, even though Apple's docs say PNG. So technically not up to spec, but does work in practice.

Reply:
Did not work on my iPhone or iPad while it work on another iPhone which was a minor fix more up to date...

I think BTW it is a bad idea to use the manual uploaded logo as the Touch Touch Logo should be uploaded separatly - we do not use the Logo as Favicon as well and we should not do that but define clear fields for that.

Reply:
I agree we should allow admins to customize this logo, many instances have custom logos and it would be confusing to have the default bookwyrm.social logo show up on their homescreen.

Perhaps initially we can add a separate configuration item for the app icon logo to use here, so that admins can create the proper logo if they want to. It could default to the main logo, or to the default Bookwyrm logo -  I'm not sure whether it would be better to have an incorrect logo, or a squashed/potentially logo.

A further enhancement may be to automatically generate a PNG of the correct size (similar to generating thumbnails) if the admin doesn't provide a mobile-specific logo.